src/platform/silabs/wifi/rs911x/platform/efx32_ncp_host.c

```diff
@@ -26,11 +26,11 @@
 #include "sl_constants.h"
 #include "sl_rsi_utility.h"
 #include "sl_si91x_host_interface.h"
-#include "sl_si91x_ncp_utility.h"
 #include "sl_si91x_status.h"
+#include "sl_si91x_ncp_utility.h"
 #include "sl_status.h"
 #include "sl_wifi_constants.h"
-#include <platform/silabs/wifi/wf200/platform/spi_multiplex.h>
+#include "spi_multiplex.h"

```

Comment:
Can you rever this change and leave the previous way it was included?

Reply:
Made this change.